src/App.js

```diff
@@ -1,9 +1,34 @@
+import {useState} from 'react';
+import Header from './components/Header';
+import BodyContainer from './components/BodyContainer';
+import members_data from './data.json';
+import SearchBar from './components/SearchBar'
+import NoSearch from './components/NoSearch'
```

Comment:
can make separate sub-folders for different components & keep their .js & .css files in it.
As functionality grows components folder will become messy in this way.